bindings/pyroot/pythonizations/test/CMakeLists.txt

```diff
@@ -108,6 +108,11 @@ if (dataframe)
     endif()
 endif()
 
+# RDFDescription pythonization
+if (rdfdescription)
```

Comment:
Shouldn't this be under the `if (dataframe)` condition block above?

Reply:
yes indeed there is no `rdfdescription` cmake option so the test should go with the other rdf tests